.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { createConfig } = require('@openedx/frontend-build');
```

Comment:
[curious] If this is using the default `eslint` config from `@openedx/frontend-build` without any overrides/extending, do we need to have the `eslintrc.js` since we'll get the base ESLint config for "free" anyways?

Reply:
Possibly? Linting seems to run fine on my machine without this file. I will remove in a follow-up PR/commit.